### ISSUE TITLE
Wiki Documentation to match newly written npc-specific prefabs in Playerclass module 

### ISSUE THREAD

**Issue by katie-gao** [uchicago-cs/chiventure#1595]
Vincent, Jonathan, and I recently written npc-specific prefabs (alchemist, lord, healer, fisherman, shopkeeper, and chef) into the class_prefab.c. This is seen at issue #1490. Now, we plan to update the wiki page to display the new npc implementations. Currently, the playerclass wiki pages only have the original classes stats and abilities, and summaries. All of the new npc-specific prefab classes will be added to the class list summary, and example stats/abilities on the Wiki.

**Comment by jroven** [comment]
Added two classes onto the Player Class ~ Class List wiki page

**Comment by katie-gao** [comment]
Jonathan, Vincent, and I that have previously worked on creating new npc-specific prefabs that are in class_prefab.c. These new classes all have descriptions and stats that are detailed in the code. In this issue, we updated the wiki documentation to include all the new classes (alchemist, lord, healer, fisherman, shopkeeper, and chef) with their descriptive summaries at https://github.com/uchicago-cs/chiventure/wiki/Player-Class-~-Class-List. As seen in the documentation, we added a new npc-specific classes subsection to detail this. Also, the npc-specific classes stats are added in: https://github.com/uchicago-cs/chiventure/wiki/Player-Class-~-Example-Stats-and-Abilities.
